### DISCUSSION TITLE
[Proposal][iOS] MediaElement force play on SilentMode

### DISCUSSION THREAD

**Discussion by Suprndm** [CommunityToolkit/Maui#1092] [New Feature Discussions]
On iOS, when a sound is played, it is mute if iphone is on silent mod. 
However, there is normaly a way to force the phone to play the sound. like here [https://bitmovin.com/docs/player/faqs/how-to-let-audio-play-when-the-ios-device-is-in-silent-mode](url)

Do you think it is possible to allow this for MediaElement ?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
